sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -489,4 +491,12 @@ public static void setFeedOptionsMaxItemCount(FeedOptions feedOptions, Integer m
     public static <T> CosmosContinuablePagedFlux<T> createCosmosContinuablePagedFlux(Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> pagedFluxOptionsFluxFunction) {
         return new CosmosContinuablePagedFlux<>(pagedFluxOptionsFluxFunction);
     }
+
+    public static Mono<FeedResponse<Document>> readMany(CosmosAsyncContainer container, 
```

Comment:
Is BridgeInernal right place? May be a new targetted contract
BridgeInternal has lot of other contracts which might get attention.

Reply:
how about this class names?
1. `ReadManyItems`
2. `Experimental`
3. `SpecialQuery`

any thought? any suggestion on the class name?

Reply:
Leaving it in the BridgeInternal until a decision is taken on where it should actually go

Reply:
We moved it to ItemOperations